pyanaconda/ui/gui/spokes/installation_source.py

```diff
@@ -436,6 +436,11 @@ def __init__(self, *args, **kwargs):
         self._network_module = NETWORK.get_proxy()
         self._device_tree = STORAGE.get_proxy(DEVICE_TREE)
 
+        # connect to the Subscription module, if possible
+        self._subscription_module = None
+        if is_module_available(SUBSCRIPTION):
+            self._subscription_module = SUBSCRIPTION.get_proxy()
```

Comment:
You use this proxy only in the `refresh` method. Otherwise, you always create a new one.

Reply:
Oh, right - I wonder why, maybe some historical reasons ? Anyway, lets use the cached module, is available.